tests/Platforms/SQLServerPlatformTestCase.php

```diff
@@ -1300,7 +1304,7 @@ public static function getGeneratesIdentifierNamesInAlterTableSQL(): iterable
             // Quoted identifiers reserved keywords.
             [
                 new TableDiff(
-                    '`table`',
+                    new Table('mytable'),
```

Comment:
Is this supposed to keep the `?
```suggestion
                    new Table('`mytable`'),
```

Reply:
The table name got accidentally changed during the refactoring but apparently, it's irrelevant for the test.